RELEASES.md

```diff
@@ -21,6 +21,41 @@ To summarize: **All our ibc-go releases allow chains to communicate successfully
 
 We ensure all major releases are supported by relayers ([hermes](https://github.com/informalsystems/ibc-rs), [rly](https://github.com/strangelove-ventures/relayer) and [ts-relayer](https://github.com/confio/ts-relayer) at the moment) which can relay between the new major release and older releases. We have no plans of upgrading to an IBC protocol specification v2.0, as this would be very disruptive to the ecosystem.
 
+## Stable Release Policy
+
+A major release series continues to receive bug fixes (released as a patch release) until it reaches End Of Life. A major release series reaches End of Life 1 year after its initial release. An extension to the End of Life date is made if the next major release is not made for 6 months. In this situation, the End of Life is the date of publication for the next major release plus 6 months. 
```

Comment:
Since using lower casing for "major release" as @crodriguezvega suggested, should we also update "End of Life"?

```suggestion
A major release series continues to receive bug fixes (released as a patch release) until it reaches end of life. A major release series reaches end of life 1 year after its initial release. An extension to the end of life date is made if the next major release is not made for 6 months. In this situation, the end of life is the date of publication for the next major release plus 6 months. 
```

Reply:
maybe we should use [dashes](https://en.wikipedia.org/wiki/End-of-life_product)?

Reply:
I actually agree, it seems to be [common format](https://en.wikipedia.org/wiki/End-of-life_product)